content/kotlin/concepts/datetime/terms/DateTimePeriod/DateTimePeriod.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '.DateTimePeriod()'
+Description: 'Returns a DateTimePeriod object or a DatePeriod object (if all arguments are set to zero) to indicate the time difference between two dates.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Data Structures'
+  - 'Time'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`DateTimePeriod`** function helps to make a new `DateTimePeriod` object that tells you how much time has passed between two dates (even up to nanoseconds). If nothing is passed into the function, it returns a `DatePeriod` object by default. You can use this function to set how many years, months, days, hours, minutes, seconds, and even nanoseconds are between two dates.
+
+## Syntax
+
+```pseudo
+fun DateTimePeriod(
+    years: Int = 0,
+    months: Int = 0,
+    days: Int = 0,
+    hours: Int = 0,
+    minutes: Int = 0,
+    seconds: Int = 0,
+    nanoseconds: Long = 0
+): DateTimePeriod
+```
+
+The function takes in up to seven parameters, and it is recommended to always explicitly name the arguments when constructing the function, like `DateTimePeriod(years = 1, months = 12)`. If all the time components are set to zero, the function will return a `DatePeriod` object.
+
+The function is also streamlined for human readability; for example, `DateTimePeriod(months = 24)` becomes `DateTimePeriod(years = 2)`.
+
+Note that the function will throw an `IllegalArgumentException` if any of the inputs overflows a Long.
```

Comment:
Add it as a Note, take reference from this [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md).